webapp/src/shared/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,61 @@
+import { Color } from '../../types/colors';
+import { ReactComponent as ArrowBack } from './assets/icons/arrow-back.svg';
+import { ReactComponent as ArrowForward } from './assets/icons/arrow-forward.svg';
+import { ReactComponent as Chat } from './assets/icons/chat.svg';
+import { ReactComponent as Edit } from './assets/icons/edit.svg';
+import { ReactComponent as File } from './assets/icons/file.svg';
+import { ReactComponent as Login } from './assets/icons/login.svg';
+import { ReactComponent as Logout } from './assets/icons/logout.svg';
+import { ReactComponent as Play } from './assets/icons/play.svg';
+import { ReactComponent as Search } from './assets/icons/search.svg';
+import { ReactComponent as Users } from './assets/icons/users.svg';
```

Comment:
I guess we cannot pack all of these imports into a barrel for consistency, because all of them have the same name and they are svgs?

Reply:
Done :+1: